package.json

```diff
@@ -36,6 +39,7 @@
     "fs-extra": "^8.1.0",
     "globby": "^10.0.1",
     "jest": "^24.9.0",
+    "jest-extended": "^0.11.2",
```

Comment:
What is this used for?

Reply:
Simplified testing. e. g. `expect(...).toIncludeAllMembers`, `expect(...). toStartWith`, `expect(...). toEndWith`.